Resources/Maps/_Goobstation/marathon.yml

```
@@ -116752,388 +116948,2400 @@ entities:
+    components:
+    - type: Transform
+      pos: -14.5,-29.5
+      parent: 30
+  - uid: 22958
+    components:
+    - type: Transform
+      pos: -14.5,-28.5
+      parent: 30
+- proto: SpawnMechRipley
+  entities:
+  - uid: 5062
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)